### ISSUE TITLE
Update dependency to support Rails 7.0.0.rc1

### ISSUE THREAD

**Issue by bruno-berchielli** [lynndylanhurley/devise_token_auth#1515]
Hello, could please update de depency of this gem to allow update to Rails 7.0.0.rc1?

```
Bundler could not find compatible versions for gem "rails":
  In Gemfile:
    rails (~> 7.0.0.rc1)

    devise_token_auth was resolved to 1.2.0, which depends on
      rails (>= 4.2.0, < 6.2)
```

Currently fetching directly from GH but still does not work. 
`
gem "devise_token_auth", git: "https://github.com/lynndylanhurley/devise_token_auth.git", branch: "master"`

Thanks!

**Comment by dhekimian** [comment]
Rails 7.0.0.rc2 and 7.0.0.rc3 have already been released in the past few days with 7.0.0 GA coming in the next week or so.

https://weblog.rubyonrails.org/2021/12/11/this-week-in-rails-rails-7-0-rc1-faster-template-rendering-better-tests-and-much-more-401a9df5/

> With tons of new features and improvements like new ways to deal with JavaScript or at-work encryption, this new version of Rails should be released before Christmas!

It might be a good idea to get this bumped & merged before more people come looking for it.

**Comment by drougojrom** [comment]
Would be great if you could push it soon! We're trying to migrate to rails 7 now, but got stuck 😅

**Comment by agildav** [comment]
👀

**Comment by agildav** [comment]
There's a PR already waiting to be merged https://github.com/lynndylanhurley/devise_token_auth/pull/1517

**Comment by booleanbetrayal** [comment]
Closed via #1517.
